mage/template.go

```diff
@@ -108,3 +108,48 @@ func list() error {
 }
 
 `
+
+var mageTpl = `// +build mage
+
+package main
+
+import (
+	"os/exec"
+	"fmt"
```

Comment:
Super nitpicky, and I'm sorry, but can you ensure this file is go-formatted?  The imports should be sorted, for example.  I know it's a pain, but I don't want the first thing that happens when someone opens the initial magefile is for their editor to reformat it.